mypy/test/helpers.py

```diff
@@ -372,16 +373,19 @@ def assert_type(typ: type, value: object) -> None:
 def parse_options(program_text: str, testcase: DataDrivenTestCase,
                   incremental_step: int) -> Options:
     """Parse comments like '# flags: --foo' in a test case."""
-    options = Options()
     flags = re.search('# flags: (.*)$', program_text, flags=re.MULTILINE)
     if incremental_step > 1:
         flags2 = re.search('# flags{}: (.*)$'.format(incremental_step), program_text,
                            flags=re.MULTILINE)
         if flags2:
             flags = flags2
 
+    disabled_error_codes = {codes.IMPLICIT_BOOL}
```

Comment:
Too many fixtures would need multiple changes to add `__len__` to list, `__bool__` to int, etc. and I understand that making fixtures more "complete" across the board is discouraged (to keep tests lean).